zetaclient/chains/bitcoin/client/client_test.go

```diff
@@ -52,6 +54,53 @@ func TestClientLive(t *testing.T) {
 		RPCParams: "testnet4",
 	}
 
+	t.Run("Get TSS top-20 UTXOs balance", func(t *testing.T) {
+		// ARRANGE
+		ts := newTestSuite(t, mainnetConfig)
+
+		// set chain
+		chain := chains.BitcoinMainnet
+		tssAddrStr := testutils.TSSAddressBTCMainnet
+		tssAddress, err := chains.DecodeBtcAddress(tssAddrStr, chain.ChainId)
+		require.NoError(t, err)
+
+		// ACT
+		ctx := context.Background()
+		utxos, err := ts.Client.ListUnspentMinMaxAddresses(ctx, 0, 9999999, []btcutil.Address{tssAddress})
+		require.NoError(t, err)
+
+		// sort utxos by amount, txid, vout descending
+		sort.SliceStable(utxos, func(i, j int) bool {
+			if utxos[i].Amount == utxos[j].Amount {
+				if utxos[i].TxID == utxos[j].TxID {
+					return utxos[i].Vout > utxos[j].Vout
+				}
+				return utxos[i].TxID > utxos[j].TxID
+			}
+			return utxos[i].Amount > utxos[j].Amount
+		})
+
+		// sum up utxos
+		var (
+			total      = 0.0
+			totalTop20 = 0.0
+			totalCount = len(utxos)
+		)
+
+		for i := range totalCount {
+			total += utxos[i].Amount
+			if i < 20 {
+				totalTop20 += utxos[i].Amount
+			}
+		}
+
+		// DISPLAY
+		fmt.Printf("utxos: %d\n", len(utxos))
```

Comment:
Might be a good idea to make this part of this issue https://github.com/zeta-chain/node/issues/4457